core/src/04-feats/01-chapter-four.md

```diff
@@ -272,7 +271,7 @@ ___
 - 
 - 
  - 
-   - **If the chosen boon has a duration of "sustain persists"**, you gain advantage 4 on your action roll to  invoke if you are not single-targeting. Additionally, one instance of the boon can be sustained each round as a free action, rather than a minor action. If the boon is somehow temporarily canceled, in the absence of other rules, you can invoke it again as a free action. If targeted by the _nullify_ bane, this effect can only be canceled by a power level 6 or greater invocation of that bane.
+   - **If the chosen boon has a duration of "sustain persists"**, you gain advantage 4 on your action roll to  invoke if you are not single-targeting. Additionally, one instance of the boon can be sustained each round as a free action, rather than a minor action. If the boon is somehow temporarily cancelled, in the absence of other rules, you can invoke it again as a free action. If targeted by the _nullify_ bane, this effect can only be canceled by a power level 6 or greater invocation of that bane.
```

Comment:
You are right, I think this was reverted from 3c5a3d610c4483f3e3c981aae307d29bd76685fc.